userspace/libpman/src/capture.c

```diff
@@ -135,6 +137,7 @@ int pman_get_scap_stats(void *scap_stats_struct)
 	struct scap_stats *stats = (struct scap_stats *)scap_stats_struct;
 	char error_message[MAX_ERROR_MESSAGE_LEN];
 	struct counter_map cnt_map;
+	int ret;
```

Comment:
This isn't used anywhere...?

Reply:
Seems a leftover yes, I'll wait for what else you may find and then push a new commit